Sources/Maker.swift

```diff
@@ -795,6 +795,23 @@ public final class Maker {
                 .centerY(to: RelationView<VerticalRelation>(view: view, relation: .centerY))
     }
 
+    /// Creates center relation rotated around center of a specified view.
+    ///
+    /// Use this method when you want to center view by both axis relativity another view.
+    ///
+    /// - parameter view: The view on which you set center relation.
+    /// - parameter radius: Radius of the arc on which center point will be placed.
+    /// - parameter angle: Angle at which center point will be placed.
+    ///
+    /// - returns: `Maker` instance for chaining relations.
+
+    @discardableResult public func center(to view: UIView, radius: CGFloat, angle: CGFloat) -> Maker {
```

Comment:
May be use separate `rotate` method instead of this one?

Reply:
`rotate` definitely won't be a better fit, since we're not rotating view here, so it won't be equally (if not more) confusing. `center` is used, because we're creating a center relation.
I'm open for suggestions here.